### DISCUSSION TITLE
Linking to numbered chapters from within an unnumbered chapters

### DISCUSSION THREAD

**Discussion by smsaladi** [quarto-dev/quarto-cli#9771] [Q&A]
### Description

There's been some discussion about unnumbered chapters and cross referencing other resources from within them:
https://github.com/quarto-dev/quarto-cli/pull/8550

There was also a merged PR that allows for cross referencing non-chapter resources from within unnumbered chapters:
https://github.com/quarto-dev/quarto-cli/pull/8550

Being able to link to numbered chapters from within an unnumbered chapter would be very helpful as well and would like to suggest this as a future addition.

**Comment by cderv** [comment]
As discussed in https://github.com/quarto-dev/quarto-cli/issues/8514 please do provide examples of what you tried and do not work for you. 

As I commented there already (https://github.com/quarto-dev/quarto-cli/issues/8514#issuecomment-2132833482), it works for me to cross ref in Quarto books.

**Comment by smsaladi** [comment]
@cderv thanks for your message. I'm going to link to your comment/screenshot here to better explain my original comment. My apologies for the lack of clarity.

              

> @smsaladi 
> 
> I can't reproduce a quarto book with what you are describing. 
> ![image](https://github.com/quarto-dev/quarto-cli/assets/6791940/20449288-381e-4338-a4a9-e0d8779f4190)
> 
> Here `#sec-intro` has been added to Intro chapter, and can be reference in Summary chapter, which has been made unnumbered  (`# Summary  {.unnumbered}`). This is with template book  `quarto create project book`. 
> 
> If this is still a problem for you, please open a new issue or discussion thread with a reproducible example or steps of what you tried, as advised above. Preferably, use the latest prerelease version. 
> 
> Thank you !

_Originally posted by @cderv in https://github.com/quarto-dev/quarto-cli/issues/8514#issuecomment-2132833482_
            
To unpack the screenshot, Summary is an unnumbered chapter, so `@sec-intro` is rendered as "1 Introduction" when written in `summary.qmd`. If `@sec-intro` is written in a numbered chapter, then it will be rendered as "Chapter 1".

This is not always the expected functionality when a reference to a numbered chapter is used within an unnumbered chapter. For example, the Preface to a book may be unnumbered but within the Preface, it can be useful to say "Chapter 1 will talk about", "Chapter 2 will talk about", "Chapter 3 will talk about"

**Reply by cderv** [reply]
Thanks that clarifies what you meant then. I wonder if we have already some issue to track this.

**Comment by adminlove520** [comment]
????????????????,?????????--???????!

????????,???????????????,??????????,???????????????????,?????????????????????

????,????????????????--??????????,????????????

?? ??
- ?? ???
